.changeset/wise-pumas-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/data-grid": patch
+---
+
+Add aria-selected to selected grid rows
```

Comment:
This changeset covers what the change was but not really how it affects the user.

We should change it to something like:

```suggestion
Fixed row selection not being announced by screenreaders
```

Reply:
👍